package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "refract-query",
+  "version": "0.0.1",
+  "description": "",
+  "main": "./lib",
+  "scripts": {
+    "prepublish": "not-in-install && npm run build || :",
+    "lint": "eslint src test",
+    "build": "babel src/ --out-dir lib/",
+    "test": "npm run lint && mocha --compilers js:babel-core/register -R spec --recursive"
```

Comment:
Yes we can definitely use [Peasant](https://github.com/danielgtaylor/peasant) here :+1: